app/helpers/case_helper.rb

```diff
@@ -1,10 +1,10 @@
 module CaseHelper
 	def uploaded_documents
-		docs=""
-       @case.documents.each do |doc|
-        docs+="* #{doc.doc_file_name}<br>"
-       end
-       docs+="No Documents Uploaded " if @case.documents.count==0 
-       return docs.html_safe
+		docs = ""
+    @case.documents.each do |doc|
+      docs += "* #{doc.doc_file_name}<br>"
+    end
+    docs += "No Documents Uploaded " if @case.documents.count == 0 
```

Comment:
No need to check `count`. Please use `@case.documents..present?` or `@case.documents.any?`. Fix this.